lib/ansible/plugins/loader.py

```diff
@@ -857,7 +857,18 @@ def _update_object(self, obj, name, path, redirected_names=None, resolved=None):
         setattr(obj, 'ansible_name', names[0])
 
     def get(self, name, *args, **kwargs):
-        return self.get_with_context(name, *args, **kwargs).object
+        ctx = self.get_with_context(name, *args, **kwargs)
+        is_core_plugin = ctx.plugin_load_context.plugin_resolved_collection == 'ansible.builtin'
+        if self.class_name == 'StrategyModule' and not is_core_plugin:
+            display.deprecated(
+                (
+                    'Use of strategy plugins not included in ansible.builtin is deprecated. No alternative '
+                    'for third party strategy plugins is currently planned.'
+                ),
+                version='2.21'
```

Comment:
Is this too fast? Some believe it is. I want to balance removing this, with also giving time. I've also been thinking this should be a "this will be removed sometime after this version without warning, so please stop using it before that version"

Thoughts?

Reply:
Agreed with bcoca, make it unbound so we can gather more info about its removal and use that to shape how we want to end up removing it. At least the unbound deprecation will publicly state that we don't support this and to not write anything new with it.